### ISSUE TITLE
Parallax extension does not cover player area

### ISSUE THREAD

**Issue by willianholtz** [4ian/GDevelop#4047]
Using Parallax Extension, and you go outside the box on the screen, the image that has parallax is not shown.

[Example here](https://github.com/4ian/GDevelop/files/8931510/My.project96.zip)

Well, actually I don't know how to report this, but I'll leave an example here to show what happens

![Screenshot_20220617_190008](https://user-images.githubusercontent.com/23488207/174406237-00d3cad4-d2b4-49a3-8480-b038efdb12c2.png)

Realize that the image is all over that place on the screen, but when running, it does not appear.

https://user-images.githubusercontent.com/23488207/174406410-5a6baf63-fe7c-4af9-bb8c-c435b73f4d4c.mp4

If this should happen, I think it needs a better description for any misunderstandings about the extension.

**Comment by Silver-Streak** [comment]
As far as I know the parallax object is supposed to be on its own layer with no camera events for that layer, so it isn't by any base layer camera movement.

It seems like you are leaving it on the main layer.

**Comment by willianholtz** [comment]
Well, according to the extension, if you leave it blank, it's the base layer that will work!

![Screenshot_20220618_123744](https://user-images.githubusercontent.com/23488207/174445749-7bc1d802-af2e-42cc-88d2-b4138d24bc45.png)

**Comment by Silver-Streak** [comment]
Yes, that's the layer the object follows with it's scrolling, basically it's "virtual camera" for it to move the background along with. Not the layer that it follows with movement of the object itself. 

You'd still want it to be on a separate layer that doesn't move.

**Comment by willianholtz** [comment]
I made an example easier to understand.

Without the extension enabled, and then with the extension enabled!
https://youtu.be/SscD9OznZgM


PS: Sorry for the delay in loading, there are already two versions that look like this and I still haven't figured out why it takes so long!

**Comment by Silver-Streak** [comment]
I'm not sure what I'm supposed to be looking for in the example in this instance. However while the information I provided above appears to be correct (looking at the old example platformer that used it, the background has to be on its own separate layer that doesn't have a moving camera, and the extension scrolls the X offset), if it isn't working for you I definitely think it's worth digging into by the author.

Unfortunately, community extensions aren't supported in the main github. It might be worthwhile to open up the issue with your example (and a description of what you're expecting vs seeing in the example) in the Extension Github:  https://github.com/GDevelopApp/GDevelop-extensions/issues
